src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/JenkinsService.java

```diff
@@ -156,6 +160,54 @@ public Optional<String> getWebHookUrl(String projectKey, String buildPlanId) {
         return Optional.of(jenkinsInternalUrlService.toInternalCiUrl(urlString));
     }
 
+    @Override
+    public void extractAndPersistBuildLogStatistics(ProgrammingSubmission programmingSubmission, ProgrammingLanguage programmingLanguage, ProjectType projectType,
+            List<BuildLogEntry> buildLogEntries) {
+        if (buildLogEntries.isEmpty()) {
+            // No logs received -> Do nothing
+            return;
+        }
+
+        if (programmingLanguage != ProgrammingLanguage.JAVA) {
+            // Not supported -> Do nothing
```

Comment:
Maybe we want to log this ? I'm not sure

Reply:
I would propose to not log it, as this function is called for every programming result and thus it is expected to do nothing, when e.g. a python exercise was submitted.

Reply:
Idea was more a debug log. But I agree it's not needed.